### PR TITLE
Support Bulk update and push from multiple IPX entries

### PR DESCRIPTION
**Pull request by sebastienwarin**
Hi @Aohzan

Thank you for this integration.

I would like to make my contribution to add two modifications:

- First, I added a new API endpoint named "IpxRequestBulkUpdateView" to push the status of all relays, digital inputs or virtuals in/out with a single call from the IPX using the labels $R, $D, $VI , etc.

- Second, I have 3 IPXs in my configuration but there is only one unique URI to push from the IPX. Therefore, only the last IPX entry in the configuration can work, the others fall into the error "API call not coming from IPX800 IP". So I added an "extra_url" for each views in order to pass in the URL the name of the IPX which makes the request.
This is an additional URL, the main URL remains the same, the code is backwards compatible with existing configurations.

Best,
Sébastien

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,12 +244,34 @@ In `URL ON` and `URL_OFF` set `/api/ipx800v4/entity_id/state`:
...
 
 ![PUSH data configuration example](ipx800_push_data_configuration_example.jpg)
 
+Finally, you can also push the states of all IPX entities directly and without naming them using bulk update. For example to update all relays from the IPX800v4 : `/api/ipx800v4_bulk/relay/$R`.
+
+![PUSH bulk configuration example](ipx800_push_bulk_configuration_example.jpg)
+
+The labels tested are as follows:
+
+-  Relays: `/api/ipx800v4_bulk/relay/$R`
```
Comment (Aohzan) [inline README.md:259]:
```suggestion
- Relays: `/api/ipx800v4_bulk/relay/$R`
```

**Review by Aohzan — APPROVED** [review]
Super merci beaucoup
On est ok qu'il n'y a aucun problème de retrocompat du coup ?

**Comment by sebastienwarin** [comment]
Oui tout à fait, on ne fait que ajouter une extra_url par vues, donc l'URL original fonctionne toujours donc 100% retrocompatible.

**Comment by Aohzan** [comment]
Merci !

**Pull request merged**